app/assets/stylesheets/shared/layout.scss

```diff
@@ -24,5 +24,4 @@ h3 {
 .prx-main {
   position: relative;
   display: grid;
-  flex-grow: 1;
```

Comment:
This was causing equal spacing between the header on the podcast index page and the posts list when there weren't enough items to fill the space.